pkg/cli/interactive_tests/test_missing_log_output.tcl

```diff
@@ -63,6 +63,10 @@ eexpect ":/# "
 
 # Start a regular server
 send "$argv start >/dev/null 2>&1 & sleep 1\r"
+eexpect "\[1\]"
+eexpect ":/# "
+# Give the server some time to start.
+sleep 0.5
```

Comment:
Anyway, this fix is still better than no fix, and I certainly didn't intend to let the perfect become the enemy of the good.